### PR TITLE
Improve Python package set

### PR DESCRIPTION
**Pull request by bcdarwin**
Minor cleanup to overrides and included packages.

### CODE REVIEW

File: container/dependencies.nix [hunk 1]
```diff
@@ -193,13 +190,15 @@ let patched_rPackages =
         testpath
         text-unidecode
         pytorchWithCuda
-        pytorch-lightning
+        #pytorch-lightning
```
Comment (cfhammill) [inline container/dependencies.nix:193]: why is lightning removed?

File: container/dependencies.nix [hunk 2]
```diff
@@ -85,7 +85,6 @@ let patched_rPackages =
         importlib-metadata
         ipykernel
         ipython
-        ipython_genutils
```
Comment (cfhammill) [inline container/dependencies.nix:88]: why has this package been removed? I don't know what it's for so I'm not sure if it can be removed

Comment (bcdarwin) [reply container/dependencies.nix:88]: It's an "internals"-type package meant to be used only by ipython and similar, so it's pulled in as a dependency and hence isn't actually being removed, but users shouldn't depend on it.

File: container/dependencies.nix (hunk 1 above)
Comment (bcdarwin) [reply container/dependencies.nix:193]: accidental, will revert.

**Review by cfhammill — APPROVED** [review]

**Pull request merged**